extensions/wikia/VideoHandlers/apiwrappers/ApiWrapper.class.php

```diff
@@ -636,9 +635,10 @@ protected function getInterfaceObjectFromType( $type ) {
 		// override me!
 	}
 
-	protected function processResponse( $response, $type ) {
+	protected function processResponse( $response ) {
```

Comment:
Minor one: let's remove the method body - this will "force" all inheriting classes to define this method - PHP Storm will even help you here :)